test/hotspot/jtreg/compiler/valhalla/inlinetypes/InlineTypes.java

```diff
@@ -140,9 +143,9 @@ public static TestFramework getFramework() {
         protected static final String UNHANDLED_TRAP = START + "CallStaticJava" + MID + "uncommon_trap.*unhandled" + END;
         protected static final String PREDICATE_TRAP = START + "CallStaticJava" + MID + "uncommon_trap.*predicate" + END;
         protected static final String MEMBAR = START + "MemBar" + MID + END;
-        protected static final String CHECKCAST_ARRAY = "(cmp.*precise klass \\[(L|Q)compiler/valhalla/inlinetypes/MyValue.*" + END;
-        protected static final String CHECKCAST_ARRAYCOPY = "(.*call_leaf_nofp,runtime  checkcast_arraycopy.*" + END;
-        protected static final String JLONG_ARRAYCOPY = "(.*call_leaf_nofp,runtime  jlong_disjoint_arraycopy.*" + END;
+        protected static final String CHECKCAST_ARRAY = "(((?i:cmp|CLFI|CLR).*" + MYVALUE_KLASS + ".*;:|.*(?i:mov|or).*" + MYVALUE_KLASS + ".*;:.*\\R.*(cmp|CMP|CLR))" + END;
```

Comment:
`CLFI` and `CLR` is only used for s390 and PPC, respectively. But I guess it does not hurt to keep it that way.